crates/forge/tests/e2e/forking.rs

```diff
@@ -0,0 +1,40 @@
+use forge::running::CACHE_DIR_NAME;
+use indoc::indoc;
+
+use crate::e2e::common::runner::{runner, setup_package};
+
+#[test]
+fn forking_and_cache() {
+    let temp = setup_package("forking");
+    let snapbox = runner();
+
+    snapbox
+        .current_dir(&temp)
+        .assert()
+        .success()
+        .stdout_matches(indoc! {r#"
+        [..]Compiling[..]
+        [..]Finished[..]
+
+
+        Collected 4 test(s) from forking package
+        Running 0 test(s) from src/
+        Running 4 test(s) from tests/
+        [PASS] tests::test_forking::latest
+        Block number: [..], block hash: [..]
+        [PASS] tests::test_forking::pending
+        Block parent hash: [..]
```

Comment:
I wonder if naming it "block parent" is clear enough. Maybe just "previous block"?

Reply:
I used the name from blockifier, but can change it